internal/checker/terraform/rules/azure/azurerm_virtual_machine_managed_disks_utilized.rego

```diff
@@ -0,0 +1,51 @@
+# METADATA
+# title: "Guarantee that Virtual Machines are making use of Managed Disks while maintaining security and compliance"
+# description: "Verify that Virtual Machines are configured to employ Managed Disks, enhancing security and compliance."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/virtual_machine
+# custom:
+#   id: CB_TFAZR_220
+#   severity: LOW
+package lib.terraform.CB_TFAZR_220
+
+import future.keywords.in
+
+isvalid(block) {
+	block.Type == "resource"
+	"azurerm_virtual_machine" in block.Labels
+}
+
+has_attribute(key, value) {
+	_ = key[value]
+}
+
+pass[block] {
+	some block in input
+	isvalid(block)
+	some innerBlock in block.Blocks
+	innerBlock.Type == "storage_os_disk"
+	has_attribute(innerBlock.Attributes, "managed_disk_type")
```

Comment:
remove has_attributes function and replace with this to avoid lint violations
 "managed_disk_type" in object.keys(inner_block.Attributes)